pipeline/minio/installs/ibm/deployment-patch.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: minio
+  annotations:
+    sidecar.istio.io/inject: "false"
```

Comment:
For multi-user, the Kubeflow namespace needs to have sidecar injection. However, Minio is not suppose to have sidecar and this was done as part of the single ibm stack deployment. However, with the new independent application approach, I need to remove all the dependencies between each application. I put the annotations to our Minio overlays so that it can be deployed after the sidecar injection labels being patched in the previous applications.